bin/reth/src/cli.rs

```diff
@@ -117,11 +120,15 @@ pub struct Logs {
     /// The filter to use for logs written to the log file.
     #[arg(long = "log.filter", value_name = "FILTER", global = true, default_value = "debug")]
     filter: String,
+
+    /// The flag to disable persistent logs.
+    #[arg(long = "log.disable-persistent", global = true, conflicts_with = "log_directory")]
+    disable_persistent: bool,
```

Comment:
why is this enabled by default?
I didn't even know this was a thing

Reply:
idk, don't mind making it disabled by default. the only reason i knew about this is because i kept running out of disk space

Reply:
same here, didn't attribute it at first to gigantic logs.

I'm for making this opt in.